src/chainstate/stacks/boot/contract_tests.rs

```diff
@@ -70,7 +70,7 @@ lazy_static! {
     )
     .unwrap();
     static ref LIQUID_SUPPLY: u128 = USTX_PER_HOLDER * (POX_ADDRS.len() as u128);
-    static ref MIN_THRESHOLD: u128 = *LIQUID_SUPPLY / 480;
+    static ref MIN_THRESHOLD: u128 = *LIQUID_SUPPLY / 8000;
```

Comment:
Could use const `TESTNET_STACKING_THRESHOLD_25` instead of 8000

Reply:
If we are fine hard coding this in our contracts, then hard code once in our tests sounds acceptable :)